client/src/sound/SoundManager.cpp

```diff
@@ -73,4 +73,8 @@ void sound::SoundManager::jumpSound() {
 	bgm.pause();
 	jump_sound.play();
 	bgm.play();
+}
+
+void sound::SoundManager::muteBgmToggle() {

```

Comment:
I think we can do toggle here: check sound, if no sound, turn on sound, or vice versa